### DISCUSSION TITLE
감사합니다

### DISCUSSION THREAD

**Discussion by 1q2w3e4r-1** [changdoc/naver-vaccine-macro#211] [General]
이틀전에 카카오로 성공해서 화이자 맞고 왔습니다
카카오로 확인하고 네이버로 예약하는걸로 했고요
다만 이틀사이에 카카오쪽이 뭔가 좀 바뀐것같아서 지금도 확실하게 되는지는 모르겠지만 아마 될거라고 생각합니다
개인적으로 처음에 한번 예약페이지 까지 갔다가 늦어서 예약이 안되어서 약간 느리다고 생각해서 map_kakao.js에서 RELOAD_INTERVAL_MILLISECONDS 부분을 500에서 250으로 바꿔서 사용했고 그 후 첫번째 시도만에 성공했습니다
개발자분에게 얘기도 없이 수정해서 사용한점 죄송합니다. 그리고 감사합니다

**Comment by jh8181** [comment]
제가 지금 예약페이지까지 갔다가 늦어서 예약이 안되는데 500에서 250으로 어떻게 설정하나요... 가르쳐 주세요. 일주일째 계속 같은 반응이에요..ㅜ

**Reply by rladbssam317** [reply]
저도 500에서 250하는법 알려주실 수 있나요?ㅠㅠ

**Reply by changdoc** [reply]
https://github.com/changdoc/naver-vaccine-macro/blob/main/map_kakao.js
4번째 줄에 500을 250으로 변경하시면 됩니다.

코드를 받아서 수행중이시라면 map_kakao.js 파일 여셔서 변경하시고 확장 프로그램을 다시 로드하시면 변경된 시간으로 동작할거에요
